app/routes/admin/tenants.py

```diff
@@ -58,12 +58,58 @@
 )
 
 
-@router.post("", response_model=CreateTenantResponse)
+@router.post("", response_model=CreateTenantResponse, summary="Create a new Tenant")
 async def create_tenant(
     body: CreateTenantRequest,
     admin_auth: AcaPyAuthVerified = Depends(acapy_auth_tenant_admin),
 ) -> CreateTenantResponse:
-    """Create a new tenant."""
+    """
+    Create a new tenant
+    ---
+
+    Use this endpoint to create a new tenant.
+    This will create a new wallet (the tenant) and onboard it with the requested roles.
+
+    The `wallet_name` is an optional field that allows you to assign a name to the wallet,
+    useful with `get_tenants` to fetch wallets by wallet name.
+
+    `image_url` is an optional field that allows tenant-admin to assign an image to the wallet.
+
+    `group_id` is an optional field that allows the tenant-admin to assign the wallet to a group.
+
+    The roles can be `issuer` or `verifier`. If no roles are provided, the tenant will be created
+    without any roles and be considered a `holder`.
+
+    `extra_settings` is an optional field that allows you to configure the wallet behaviour for advanced users.
+
+    Request body:
+    ---
+        body: CreateTenantRequest
+            wallet_label: str
+                A required alias for the tenant, publicized to other tenants when forming a connection.
+            wallet_name: Optional[str]
+                An optional wallet name. Useful with `get_tenants` to fetch wallets by wallet name.
+            roles: Optional[List[str]]
+                A list of roles to assign to the tenant.
+            image_url: Optional[str]
+                An optional image URL for the tenant.
+            group_id: Optional[str]
+                An optional group identifier.
+            extra_settings: Optional[Dict[str, Union[bool, str]]]
+                Optional per-tenant settings to configure wallet behaviour for advanced users.
```

Comment:
Ask GPT to proof read / improve some of this Maximising conciseness. We're duplicating a lot of info from the request body descriptions. We should just add noteworthy things here (no roles = regular holder. image_url is added to the trust registry if they have a role). And I don't think we should refer to `tenant-admin` here, since the reader will presumably be an admin.